docs/data-sources/appliances.md

```diff
@@ -0,0 +1,39 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spectrocloud_appliances Data Source - terraform-provider-spectrocloud"
+subcategory: ""
+description: |-
+  
+---
+
+# spectrocloud_appliances (Data Source)
+
+
+
+## Example Usage
+
+```terraform
+data "spectrocloud_appliances" "appliances" {
+  #tags = ["env:prod", "store:502"]
+  tags = ["env:dev"]
+}
+
+output "same" {
+  value = data.spectrocloud_appliances.appliances
+  #value = [for a in data.spectrocloud_appliance.appliances : a.name]
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `tags` (Set of String)
```

Comment:
If lookup is done by tags, shouldn't they be mandatory to ensure uniqueness?

Reply:
Also, if we're switching to lookups by tags, we should update the description in the appliance schema to document this behaviour.

Reply:
I'm considering changing it to a map. Will make the corresponding change later this week. PLT-152 ticket has already been opened for it.